views/play.erb

```diff
@@ -0,0 +1,19 @@
+<h2>Welcome <%= @name %> </h2><br><br>
+
+<h1> ROCK, PAPER OR SCISSORS? </h1>
+      <br><br>
+
+
+
+
+<form action="/result" method="post">
+  <div>
+    <label for="Rock">Rock</label>
+    <input type="radio" name="move" value="Rock">
+    <label for="Paper">Paper</label>
+    <input type="radio" name="move" value="Paper">
+    <label for="Scissor">Scissor</label>
+    <input type="radio" name="move" value="Scissor">
+    <button type="submit" value="submit"><%= @name %>'s move</button>
```

Comment:
Using interpolation here, making the submit button read nicely with the dynamic data